docs/integrate/etl/kafka-connect.rst

```diff
@@ -1,504 +0,0 @@
-
-.. code-block:: json
-
-   {
-     "namespace": "cratedb.metrics",
-     "name": "value",
-     "type": "record",
-     "fields": [
-       {"name": "id", "type": "string"},
-       {"name": "timestamp", "type": "float"},
-       {"name": "payload", "type": {
-           "type": "record",
-           "name": "payload",
-           "fields": [
-             {"name": "temperature", "type": "float"},
-             {"name": "humidity", "type": "float"},
-             {"name": "pressure", "type": "float"},
-             {"name": "luminosity", "type": "float"}
-           ]
-         }
-       }
-     ]
-   }
-
-
-For this example, this Python script will simulate the creation
-of random sensor data and push it into the ``metrics`` topic:
-
-.. code-block:: python
-
-   import time
-   import random
-
-   from confluent_kafka import avro
-   from confluent_kafka.avro import AvroProducer
-
-   # Define the Avro schema we want our produced records to conform to.
-   VALUE_SCHEMA_STR = """
-   {
-     "namespace": "cratedb.metrics",
-     "name": "value",
-     "type": "record",
-     "fields": [
-       {"name": "id", "type": "string"},
-       {"name": "timestamp", "type": "float"},
-       {"name": "payload", "type": {
-           "type": "record",
-           "name": "payload",
-           "fields": [
-             {"name": "temperature", "type": "float"},
-             {"name": "humidity", "type": "float"},
-             {"name": "pressure", "type": "float"},
-             {"name": "luminosity", "type": "float"}
-           ]
-         }
-       }
-     ]
-   }
-   """
-
-   # Load the Avro schema.
-   VALUE_SCHEMA = avro.loads(VALUE_SCHEMA_STR)
-
-   # Create an Avro producer using the defined schema, assuming that our
-   # Kafka servers are running at localhost:9092 and the Schema Registry
-   # server is running at localhost:8081.
-   AVRO_PRODUCER = AvroProducer(
-       {
-           "bootstrap.servers": "localhost:9092",
-           "schema.registry.url": "http://localhost:8081",
-       },
-       default_value_schema=VALUE_SCHEMA,
-   )
-
-   # Create a metric payload from a simulated sensor device.
-   def create_metric():
-       return {
-           "id": "sensor-" + str(random.choice(list(range(1, 21)))),
-           "timestamp": int(time.time()),
-           "payload": {
-               "temperature": random.uniform(-5, 35),
-               "humidity": random.uniform(0, 100),
-               "pressure": random.uniform(1000, 1030),
-               "luminosity": random.uniform(0, 65000),
-           },
-       }
-
-   # Create a new metric every 0.25 seconds and push it to the metrics topic.
-   while True:
-       AVRO_PRODUCER.produce(topic="metrics", value=create_metric())
-       AVRO_PRODUCER.flush()
-       time.sleep(0.25)
-
-This script can be run by installing the following dependencies and running it:
-
-.. code-block:: console
-
-   $ pip install "confluent-kafka[avro]" "avro-python3"
-   $ python simulator.py
```

Comment:
I am sad to see this Python snippet removed. Maybe we can bring it back in one way or another, if not within this tutorial, maybe at another spot.

Reply:
Absolutely. The current setup is pretty crude, I'm sure we can improve it with a similar script